packages/angular_devkit/build_angular/src/utils/routes-extractor/extractor.ts

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
@AndrewKushnir and @atscott would appreciate if you could take a look at the changes in this file.